tests/functional/monitoring/prometheus/alerts/test_alerting_works.py

```diff
@@ -39,17 +40,48 @@ def test_alerting_works(threading_lock):
 def test_prometheus_rule_failures(threading_lock):
     """
     There should be no PrometheusRuleFailures alert when OCS is configured.
+    This test is extended to check for many-to-many matching errors in Prometheus logs.
```

Comment:
Please include the DFBUG number here to indicate that this scenario is added from a customer reported bug

Reply:
added, thanks